code/__HELPERS/pronouns.dm

```diff
@@ -0,0 +1,242 @@
+//pronoun procs, for getting pronouns without using the text macros that only work in certain positions
+//datums don't have gender, but most of their subtypes do!
+/datum/proc/p_they(capitalized, temp_gender)
```

Comment:
It may be best to implement this first, and change allll of the existing code to properly use this, then go about with the emotes stuff; no sense in porting over TG's pronoun system, only to never have it utilized except for emotes.

Reply:
Kay, I'll put up another PR for that. Will just have to rebase this on top of the pronouns PR when that's done.